cmd/logbridge/main.go

```diff
@@ -116,8 +116,23 @@ func startReader(ch chan<- []map[string]interface{}) error {
 	if err != nil {
 		return err
 	}
-
-	cursor, err := readCursor()
+	var cursor string
+	cursor, err = readCursor()
+	if err != nil {
```

Comment:
does not handle the startup case where the state file does not exist

Reply:
I was wondering, what created file in the first place?